webapps-schema/src/main/java/io/camunda/webapps/schema/entities/auditlog/AuditLogScope.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.auditlog;
+
+public enum AuditLogScope {
```

Comment:
On a second thought, shall we maybe call it `AuditLogTenantScope` to make it more specific?